lib/client/js/ui/controls/Menu.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Insert description of Menu class
+ *
+ * @private
```

Comment:
True - neither should be labelled as such. I missed that before. Would you mind updating that?